### PR TITLE
Allows the unsetting of the RPC endpoint

### LINKED ISSUE #986: There is no way to unset an ethereum node in the settings
## Problem Definition

At the moment if you succesfully set an ethereum node and you want to unset it you can't really do it. The application only accepts another valid ethereum node connection.

## Task

Allow removing an ethereum node connection from the frontend settings.

This should work by adding a check for the empty string right before the attempt connect call here:
https://github.com/rotki/rotki/blob/268bd118f4e4ff3d750e5968694b8b1f8e0b1145/rotkehlchen/chain/ethereum/manager.py#L165

Something like this:

```python
        if endpoint == '':
            if self.web3:
                del self.web3
            self.connected = False
            self.ethrpc_endpoint = ''
```

In the frontend there should be some nice way to send the empty string, rather than simply deleting the endpoint string.

### PR DESCRIPTION
**Pull request by kelsos**
Fixes #986

### CODE REVIEW

File: rotkehlchen/chain/ethereum/manager.py [hunk 1]
```diff
@@ -158,15 +158,23 @@ def is_synchronized(self, current_block: int, latest_block: int) -> Tuple[bool,
...
-            self.ethrpc_endpoint = endpoint
-        return result, message
+           Returns a tuple (result, message)
+               - result: Boolean for success or failure of changing the rpc endpoint
+               - message: A message containing information on what happened. Can
+                          be populated both in case of success or failure"""
+        if endpoint == '':
+            if self.web3:
+                del self.web3
+                self.web3 = None
```
Comment (kelsos) [inline rotkehlchen/chain/ethereum/manager.py:168]: Without setting to None, setting setting a valid connection would fail because of the missing property.

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/manager.py:168]: yea due to `if self.web3`

**Comment by LefterisJP** [comment]
Only 1 comment @kelsos but I think this is by design. It seems you can copy paste anything in the field and it will send it immediately to the backend. I had copied pasted the entire url except first `h` of `http` and wanted to fix it between copying and pasting and I couldn't. :smile:

**Comment by kelsos** [comment]
Yes, I had to add because it changes on the change event, and if you clear and paste it won't fire up. I clear the value if it fails but maybe we could change it to keep the wrong value since someone might want to edit it.

**Comment by isidorosp** [comment]
> Yes, I had to add because it changes on the change event, and if you clear and paste it won't fire up. I clear the value if it fails but maybe we could change it to keep the wrong value since someone might want to edit it.

You can also use @blur instead of @change

**Pull request merged**